package.json

```diff
@@ -13,7 +13,9 @@
     "url": "https://github.com/DevExpress/DevExtreme.AspNet.Data.git"
   },
   "main": "js/dx.aspnet.data.js",
+  "types": "js/dx.aspnet.data.d.ts",
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
This is what I wanted to ask! 🎓 
Is it a normal practice to face "Cannot find name 'JQueryAjaxSettings'." error and then guess to install "@types/jquery"?